src/turtle_window.rs

```diff
@@ -98,6 +99,11 @@ impl TurtleWindow {
         self.send_drawing_command(Clear);
     }
 
+    /// Save drawings as SVG
+    pub fn save_svg<P: AsRef<path::Path>>(&self, path: P) {
+        self.send_drawing_command(SaveSVG((*path.as_ref()).to_path_buf()));
```

Comment:
Per my other comment, you can change this function to take `path: &Path` as a parameter. You should be able to call `path.to_path_buf()` instead of `(*path.as_ref()).to_path_buf()`. 

(Side Note: I am pretty sure that you don't need to explicitly deref here because Rust will do that for you automatically. That means that this code could have been `path.as_ref().to_path_buf()`. You'll be replacing this code anyway so it doesn't really matter, but I thought I'd mention that.)